src/main/java/pl/pop/interview/master/question/QuestionController.java

```diff
@@ -26,6 +26,15 @@ public List<QuestionDTO> getAllQuestions() {
         return questionFacade.getAllQuestions();
     }
 
+    @GetMapping("/{id}/answers")
+    public List<QuestionDTO> getQuestionsForPractitioner(@PathVariable("id") Long id, @RequestParam(required = false) Boolean answered) {
+        if (answered != null && answered) {
```

Comment:
Logika biznesowa powinna być w fasadzie. 

Controller nie powinien tutaj mieć cokolwiek do gadania. 
On tylko przekazuje informacje dalej. 

Cały ten if, o ile potrzebny, a z tego co widzę, to nie jest potrzebny, powinien być w fasadzie.

Reply:
OK - logika przeniesiona do fasady